profiles/common/modules/features/cce_basic_config/cce_basic_config.install

```diff
@@ -2200,3 +2144,126 @@ function cce_basic_config_update_7213() {
+      'text',
+    );
+    $field_info_map = field_info_field_map();
+    $filtered = array_filter($field_info_map, function ($item) use ($field_types) {
+      return in_array($item['type'], $field_types);
+    });
+
+    $sandbox['field_list'] = array_keys($filtered);
+    $sandbox['full_html_track_used'] = FALSE;
+
+    $sandbox['max'] = count($sandbox['field_list']);
+  }
+
+  $field_name = array_shift($sandbox['field_list']);
+
+  // Check if there are tracked changes in any revision of the "text" field.
+  $query = db_select('field_revision_' . $field_name, 'fr');
+  $query->where('fr.' . $field_name . '_value REGEXP \'<span[^>]+class="[^"]*ice-[^>]+>\'')
+    ->condition('fr.deleted', 0);
+  $query->addExpression('1');
+  $query->distinct(TRUE);
+  $scanning_results = $query->execute()->fetchField();
+
+  // Check if full_html_track text format is used in any revision of the
+  // "text" field.
+  if (!$scanning_results) {
+    $query = db_select('field_revision_' . $field_name, 'fr');
+    $query->condition('fr.' . $field_name . '_format', 'full_html_track')
+      ->condition('fr.deleted', 0);
+    $query->addExpression('1');
+    $scanning_results = $query->execute()->fetchField();
+  }
+
+  if ($scanning_results) {
+    // Stop the batch because we have the minimum of info to proceed.
+    $sandbox['field_list'] = array();
+    $sandbox['max'] = 0;
+
+    module_enable(array('nexteuropa_trackedchanges'));
+
+    $ms_wysiwyg = variable_get('multisite_wysiwyg');
+    if ($ms_wysiwyg) {
+      if (isset($ms_wysiwyg['dis_change_track_on_create'])) {
+        variable_set('nexteuropa_trackedchanges_disable_track_on_create', $ms_wysiwyg['dis_change_track_on_create']);
+      }
+      if (isset($ms_wysiwyg['en_change_track_on_edit'])) {
+        variable_set('nexteuropa_trackedchanges_force_track_on_edit', $ms_wysiwyg['en_change_track_on_edit']);
+      }
+    }
+
+    $wbm_state = variable_get('nexteuropa_editorial_tracking_wbm_states');
+    if (!is_null($wbm_state)) {
+      variable_set('nexteuropa_trackedchanges_tracking_wbm_states', $wbm_state);
+    }
+
+    $status_setting = variable_get('nexteuropa_editorial_tracking_status');
+    if (!is_null($status_setting)) {
+      variable_set('nexteuropa_editorial_tracking_status', $status_setting);
+    }
+
+    $sandbox['full_html_track_used'] = TRUE;
+  }
+  $sandbox['progress']++;
+
+  // Cleaning old parameters.
+  if (empty($sandbox['max']) || ($sandbox['progress'] == $sandbox['max'])) {
+    watchdog('gilles end', $sandbox['full_html_track_used']);
+    variable_del('multisite_wysiwyg');
+    variable_del('nexteuropa_editorial_tracking_status');
+    variable_del('nexteuropa_editorial_tracking_wbm_states');
+
+    if (!$sandbox['full_html_track_used']) {
+      // If full_html_track is not used anywhere, we could delete it too.
+      multisite_config_service('wysiwyg')->deleteProfile('full_html_track');
+      // We only disable it because Drupal does not allow to delete it
```

Comment:
And I think it's good, what if they want to use it in the future if you delete it?